doc/man3/ASN1_aux_cb.pod

```diff
@@ -87,7 +87,7 @@ found for the purposes of reference counting.
 =item I<asn1_cb>
 
 A callback that will be invoked at various points during the processing of
-the the B<ASN1_VALLUE>. See below for further details.
+the B<ASN1_VALLUE>. See below for further details.
```

Comment:
Could you please also fix the obvious typo?

Reply:
Damn, didn't spot that. Will do